frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -510,6 +498,35 @@ The interval value is in seconds and can range from 3 to 20 seconds.]]),
             end,
         })
     end
+    table.insert(menu_items.long_press.sub_item_table, {
+        text_func = function()
+            return T(_("Highlight long-press interval: %1 s"),
+                G_reader_settings:readSetting("highlight_long_hold_threshold_s", 3))
+        end,
+        keep_menu_open = true,
+        callback = function(touchmenu_instance)
+            local SpinWidget = require("ui/widget/spinwidget")
+            local items = SpinWidget:new{
+                title_text = _("Highlight long-press interval"),
+                info_text = _("If a touch is not released in this interval, it is considered a long-press. On document text, single word selection will not be triggered."),
```

Comment:
As mentionned in a previous comment, this wording "long-press" here is wrong.
It should be "long long-press" or "very long press", or whatever @Frenzie can think of :)

Given that the "You are in SELECT mode" new message is more critical than this one, we can update this one in this PR.

Reply:
So waiting for the final wording.

Reply:
While I think very long press or extra long press sounds a bit silly I can't really think of anything better.

Something like `prolonged` might fit but it'd be lacking in clarity.

Reply:
"Very long press" is clear enough. 
It gives the idea that press longer than your usual "long".